requirements.txt

```diff
@@ -1,7 +1,6 @@
 netaddr
 netifaces
-enum34
-scapy
+scapy==2.4.3rc1
```

Comment:
Why are you using a release candidate and not the latest Scapy?

Reply:
There are scapy failures when building kube-hunter image with the latest scapy. These failures were fixed in the release candidate.